src/components/Projects.tsx

```diff
@@ -2,33 +2,97 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { makeStyles } from '@material-ui/core/styles';
 import { Container, Box, Paper } from '@material-ui/core';
-
-/*
-Template for future page components.
-Use MUI's CSS-in-JS makeStyles() as shown below for more complex CSS/
-Otherwise, use <Box> components where possible for inline styling.
-Every page must have a Helmet tag for SEO purposes.
-*/
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
 
 const useStyles = makeStyles({
   example: {
     color: 'red',
   },
 });
 
-const Template: React.FC = () => {
+const Project: React.FC = () => {
   const classes = useStyles();
   return (
     <Container maxWidth="md">
       <Helmet>
-        <title>Write Title Here</title>
+        <title>Code4Community - Projects</title>
         <meta name="description" content="Write a description for SEO here" />
       </Helmet>
       <Box my={4}>
         <Paper className={classes.example} />
+
+        <Card className="project-card">
+          <CardActionArea>
+            <CardMedia
+              component="img"
+              className="project-img"
+              image="./llb.png"
+              title="LLB"
```

Comment:
Getting an ESLint warning for this one
<img width="732" alt="Screen Shot 2020-08-02 at 8 35 38 AM" src="https://user-images.githubusercontent.com/15748593/89123155-25f42c80-d49b-11ea-8ca4-8be950ace8be.png">
The alt property is required on the internal image I believe.

Reply:
strange, doesn't appear for me, but I fixed it anyways